content/en/docs/examples/bookinfo/index.md

```diff
@@ -192,22 +192,16 @@ versions, called *subsets*, in [destination rules](/docs/concepts/traffic-manage
 
 Run the following command to create default destination rules for the Bookinfo services:
 
-* If you did **not** enable mutual TLS, execute this command:
-
-    {{< tip >}}
-    Choose this option if you are new to Istio and are using the `demo`
-    [configuration profile](/docs/setup/additional-setup/config-profiles/).
-    {{< /tip >}}
-
-    {{< text bash >}}
-    $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
-    {{< /text >}}
-
-* If you **did** enable mutual TLS, execute this command:
+{{< text bash >}}
+$ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
+{{< /text >}}
 
-    {{< text bash >}}
-    $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all-mtls.yaml@
-    {{< /text >}}
+{{< tip >}}
+You can enforce mutual TLS using the `samples/bookinfo/networking/destination-rule-all-mtls.yaml`
+destination rule.  However, the `default` and `demo` [configuration profiles](/docs/setup/additional-setup/config-profiles/) have [auto mutual TLS](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls) enabled by default.  If a TLS context is not provided in a
+destination rule, Istio configures client proxies to send mutual TLS traffic to injected proxies
+automatically.
```

Comment:
The last sentence seem unnecessary given the link to the auto-mutual-tls doc in the previous sentence. Also, maybe reversing the first two sentences would flow better?
```suggestion
The `default` and `demo` [configuration profiles](/docs/setup/additional-setup/config-profiles/) have [auto mutual TLS](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls) enabled by default.
To enforce mutual TLS, use the destination rules in `samples/bookinfo/networking/destination-rule-all-mtls.yaml`.
```

Reply:
In an effort to simplify our documentation, I wonder if we really need this tip. Shouldn't users just use AuthenticationPolicy to enforce mtls in their entire namespace/mesh instead of using DR to enforce tls per service?
I recommend removing this tip. I also think we should take it a step further and just remove this whole section and then move the DR apply command as a step in the previous section. Or just move it to `Request Routing` task

Reply:
I'm trying to avoid introducing users to Custom Resources they might not need know unless they are doing the task that needs them.

Reply:
Removing the section entirely isn't a good idea IMO, because this app is deploying several versions of the review service, so defining their subsets should really be a prereq before moving on.

I think the tip is also fine since this app is used in a lot more places than just the getting started demo. I think it's much better as Greg made it a tip (easy to ignore when getting started) instead of inline noise, like it was before.